moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -97,6 +97,12 @@ PR2ArmIKSolver::PR2ArmIKSolver(const urdf::ModelInterface& robot_model, const st
   active_ = pr2_arm_ik_.init(robot_model, root_frame_name, tip_frame_name);
 }
 
+void PR2ArmIKSolver::updateInternalDataStructures()
+{
+  // TODO: move (re)allocation of any internal data structures here
```

Comment:
TODO should have a username or issue number

Reply:
This function has been removed by mistake and I just reverted it, the original TODO also exists like that in MoveIt 1. Not sure if it makes sense to change this here